concent_api/api-integration-additional-verification-test.py

```diff
@@ -88,12 +88,16 @@ def test_case_1_test_for_positive_case(cluster_consts, cluster_url, test_id):
     current_time = get_current_utc_timestamp()
     (subtask_id, task_id) = get_task_id_and_subtask_id(test_id, 'existing_file')
 
-    result_file_content_1 = task_id
-    source_file_content_2 = subtask_id
-    result_file_size_1 = len(result_file_content_1)
-    source_file_size_2 = len(source_file_content_2)
-    result_file_check_sum_1 = 'sha1:' + hashlib.sha1(result_file_content_1.encode()).hexdigest()
-    source_file_check_sum_2 = 'sha1:' + hashlib.sha1(source_file_content_2.encode()).hexdigest()
+    current_dir = os.path.dirname(os.path.abspath(__file__))
+    with open(os.path.join(current_dir, 'tests_resources', 'source.zip'), 'rb') as f:
```

Comment:
I think if we want to upload these files from our disc and store them on our repository then these files must be as small as possible. Maybe good idea would be to create small `test_resources` with correct and incorrect results?

Reply:
I will discuss it wil @cameel

Reply:
as discussed with @cameel, at the moment we will not store `test_resources` in repo.
I will remove the files

Reply:
Are we going to use hardcoded subtask_id as in other cluster tests ? Just asking with curiosity.